packages/react-components/react-dialog/src/components/DialogSurface/useDialogSurface.ts

```diff
@@ -30,6 +31,31 @@ export const useDialogSurface_unstable = (
   const requestOpenChange = useDialogContext_unstable(ctx => ctx.requestOpenChange);
   const dialogTitleID = useDialogContext_unstable(ctx => ctx.dialogTitleId);
 
+  const { findFirstFocusable } = useFocusFinders();
+
+  const hasTabIndexProps = props.hasOwnProperty('tabIndex');
+
+  const dialogRefCallback = React.useCallback(
+    (dialog: HTMLElement | null) => {
+      if (
+        dialog &&
+        findFirstFocusable(dialog) === null &&
```

Comment:
Do we need to fix this? We've only seen one case where this is an issue, and that can be worked around